README.md

```diff
@@ -10,7 +10,7 @@
 ## Multi-cluster Configuration Management for Edge, Multi-Cloud, and Hybrid Cloud
 <br/>
 
-**NOTE**: the website kubestellar.io is temporarily out of order.
+**NOTE**: Some of our documentation is on the website [kubestellar.io](kubestellar.io), some is here.
```

Comment:
Maybe:  
You can find more information on the project web site at  [kubestellar.io](kubestellar.io)  . Please note the site is still under construction.

Reply:
I have reworded this.